pro/src/app/App/analytics/sentry.ts

```diff
@@ -111,7 +111,7 @@ export const useSentry = () => {
   const currentUser = useSelector(selectCurrentUser)
 
   useEffect(() => {
-    if (currentUser !== null) {
+    if (currentUser !== null && currentUser.id) {
```

Comment:
Si c'est `undefined` tu ne vas pas avoir une autre erreur ?
```suggestion
    if (currentUser && currentUser.id) {
```

Reply:
en soit c'est pas possible, mais comme l'erreur est pas censé être possible non plus go ta solution :)